Cubical/Categories/Functor.agda

```diff
@@ -8,17 +8,18 @@ open import Cubical.Categories.Category
 
 private
   variable
-    ℓ𝒞 ℓ𝒞' ℓ𝒟 ℓ𝒟' : Level
+    ℓC ℓC' ℓD ℓD' : Level
 
-record Functor (𝒞 : Precategory ℓ𝒞 ℓ𝒞') (𝒟 : Precategory ℓ𝒟 ℓ𝒟') : Type (ℓ-max (ℓ-max ℓ𝒞 ℓ𝒞') (ℓ-max ℓ𝒟 ℓ𝒟')) where
+record Functor (C : Precategory ℓC ℓC') (D : Precategory ℓD ℓD') : Type (ℓ-max (ℓ-max ℓC ℓC') (ℓ-max ℓD ℓD')) where
   no-eta-equality
   open Precategory
 
   field
-    F-ob : 𝒞 .ob → 𝒟 .ob
-    F-hom : {x y : 𝒞 .ob} → 𝒞 .hom x y → 𝒟 .hom (F-ob x) (F-ob y)
-    F-idn : {x : 𝒞 .ob} → F-hom (𝒞 .idn x) ≡ 𝒟 .idn (F-ob x)
-    F-seq : {x y z : 𝒞 .ob} (f : 𝒞 .hom x y) (g : 𝒞 .hom y z) → F-hom (𝒞 .seq f g) ≡ 𝒟 .seq (F-hom f) (F-hom g)
+    F-ob : C .ob → D .ob
```

Comment:
How about calling these `fmapOb`, `fmapHom`, `fmapId` and `fmap⋆`?

Reply:
Can we keep these for now? I've used these a bunch in later commits already (which I'll be making PRs for soon) and also I do kind of like the current naming. (same for nat).

Reply:
Ok, the F can stand for "Functor" of course :-)